Lib/test/test_syntax.py

```diff
@@ -1988,20 +1985,29 @@ def test_return_outside_function(self):
                           "outside function")
 
     def test_break_outside_loop(self):
-        self._check_error("if 0: break",             "outside loop")
-        self._check_error("if 0: break\nelse:  x=1",  "outside loop")
-        self._check_error("if 1: pass\nelse: break", "outside loop")
-        self._check_error("class C:\n  if 0: break", "outside loop")
+        self._check_error("break", "outside loop", lineno=1)
+        self._check_error("if 0: break",             "outside loop", lineno=1)
+        self._check_error("if 0: break\nelse:  x=1",  "outside loop", lineno=1)
+        self._check_error("if 1: pass\nelse: break", "outside loop", lineno=2)
+        self._check_error("class C:\n  if 0: break", "outside loop", lineno=2)
         self._check_error("class C:\n  if 1: pass\n  else: break",
-                          "outside loop")
+                          "outside loop", lineno=3)
+        self._check_error("with object() as obj:\n break",
+                          "outside loop", lineno=2)
 
     def test_continue_outside_loop(self):
-        self._check_error("if 0: continue",             "not properly in loop")
-        self._check_error("if 0: continue\nelse:  x=1", "not properly in loop")
-        self._check_error("if 1: pass\nelse: continue", "not properly in loop")
-        self._check_error("class C:\n  if 0: continue", "not properly in loop")
+        self._check_error("if 0: continue",             "not properly in loop",
+                          lineno=1)
+        self._check_error("if 0: continue\nelse:  x=1", "not properly in loop",
+                          lineno=1)
+        self._check_error("if 1: pass\nelse: continue", "not properly in loop",
+                          lineno=2)
+        self._check_error("class C:\n  if 0: continue", "not properly in loop",
+                          lineno=2)
```

Comment:
This is a bit tidier, no?

```suggestion
        msg = "not properly in loop"
        self._check_error("if 0: continue", msg, lineno=1)
        self._check_error("if 0: continue\nelse:  x=1", msg, lineno=1)
        self._check_error("if 1: pass\nelse: continue", msg, lineno=2)
        self._check_error("class C:\n  if 0: continue", msg, lineno=2)
```

Reply:
Looks better!